en/metadata.yaml

```diff
@@ -47,6 +47,35 @@ navigation:
       - title: Advanced - Branded store
         location: guides/go-to-production/advanced.md
 
+  - title: Ubuntu Core Stacks
+
+    children:
+      - title: Introduction
+        location: stacks/index.md
+
+      - title: Stack Items
+        children:
+          - title: Audio management
+            location: stacks/audio/index.md
+          - title: Bluetooth management
+            location: stacks/bluetooth/doc/overview.md
+          - title: Disk management
+            location: stacks/disk/index.md
+          - title: Location services
```

Comment:
In the main navigation, please don't add sub-pages without a location, they clutter navigation and make you hunt for actual links, these WIP topics are already visible (and don't look like links) on the index page of the section.